gpii/node_modules/flowManager/src/BrowserChannel.js

```diff
@@ -0,0 +1,37 @@
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion");
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.request.flowManager");
+
+    fluid.defaults("gpii.flowManager.browserChannel", {
+        gradeNames: ["fluid.standardRelayComponent", "autoInit"]
+    });
+
+    fluid.defaults("kettle.requests.request.handler.browserChannel", {
+        gradeNames: ["gpii.request.flowManager.sessionAware", "autoInit"],
+        invokers: {
+            handle: {
+                funcName: "gpii.request.flowManager.browserChannelHandle",
+                args: ["{request}"],
+                dynamic: true
+            }
+        }
+    });
+
+    gpii.request.flowManager.browserChannelHandle = function (fluidRequest) {
+        var data = fluidRequest.data;
```

Comment:
Already did this [here](https://github.com/javihernandez/universal/commit/05b93274e9d23c552610b61048db5a8641450e4b#diff-1f34e9fa5d243292e63a36cb78875bceR38)